api/Window.json

```diff
@@ -50,6 +50,92 @@
           "deprecated": false
         }
       },
+      "cancelAnimationFrame": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/cancelAnimationFrame",
+          "support": {
+            "chrome": [
+              {
+                "version_added": "24"
+              },
+              {
+                "version_added": "21",
+                "version_removed": "24",
+                "prefix": "webkit"
+              }
+            ],
+            "chrome_android": [
+              {
+                "version_added": "24"
+              },
+              {
+                "version_added": "21",
+                "version_removed": "24",
+                "prefix": "webkit"
+              }
+            ],
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": [
+              {
+                "version_added": "23"
+              },
+              {
+                "version_added": "11",
+                "version_removed": "23",
+                "prefix": "moz"
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "23"
+              },
+              {
+                "version_added": true,
+                "version_removed": "23",
+                "prefix": "moz"
+              }
+            ],
+            "ie": {
+              "version_added": "10"
+            },
+            "opera": {
+              "version_added": "15"
+            },
+            "opera_android": {
+              "version_added": "33"
+            },
+            "safari": [
+              {
+                "version_added": "6.1"
+              },
+              {
+                "version_added": "6",
+                "version_removed": "6.1",
+                "prefix": "webkit"
+              }
+            ],
+            "safari_ios": {
+              "version_added": "7.1"
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            },
+            "webview_android": {
+              "version_added": true
```

Comment:
needinfo?(@jpmedley)

Reply:
Yes, I wondered if this should be the same as the Chrome version.

Reply:
Re this and previous comment: for features this far back, I don't have information that I have any confidence in. Put `true` for both.